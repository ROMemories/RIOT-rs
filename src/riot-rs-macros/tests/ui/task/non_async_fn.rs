#![no_main]
#![feature(type_alias_impl_trait)]
#![feature(used_with_arg)]

// FAIL: the function must be async
#[riot_rs::task]
fn main() {}
