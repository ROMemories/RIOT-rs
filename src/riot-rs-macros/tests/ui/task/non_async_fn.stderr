error: custom attribute panicked
 --> tests/ui/task/non_async_fn.rs:7:1
  |
7 | #[riot_rs::task]
  | ^^^^^^^^^^^^^^^^
  |
  = help: message: the function must be async
